src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import { useForm } from 'react-hook-form';
+
+import { SIGN_UP_FIELDS } from './constants';
+import styles from './styles.module.scss';
+import logo from './assets/wolox_logo.svg';
+
+function SignUp() {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm();
+
+  const fields = SIGN_UP_FIELDS;
+
+  const onSubmit = (data: any): void => {
```

Comment:
please changed any for 
`export interface UserForm {
  email: string;
  password: string;
  password_confirmation: string;
  first_name: string;
  last_name: string;
  locale: string;
}`